<?xml version="1.0"?>
<launch>

	<param name="use_sim_time" value="true"/>
	<node name="PISquare" pkg="grasp_learning" type="PISquare.py" output="screen">
 		<param name="num_inputs" value="1" />
		<param name="batch_size" value="3" />
		<param name="relative_path" value="$(find grasp_learning)/stored_data/tested_data/" />
		<param name="max_num_trials" value="5" />
		<param name="min_dist_norm" value="0.04" />
		<param name="num_samples" value="5000" />
		<param name="num_params" value="100" />
 	</node>
</launch>
